File: codeforces/low_difficulty/difficulty_1000/3.young_physicst_[69A]/YoungPhysicist.java
/* Author: José Rodolfo (jric2002) */
import java.util.Scanner;
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
public class YoungPhysicist {
  public static void main(String[] args) throws IOException{
    Short n;
    Short x, y, z;
    String[] f;
    Scanner input_n = new Scanner(System.in);
    n = input_n.nextShort();
    x = y = z = 0;
    BufferedReader input_f = new BufferedReader(new InputStreamReader(System.in));
    while (n > 0) {
      f = (input_f.readLine()).split(" ");
      x = (short)(x + Short.valueOf(f[0]));
      y = (short)(y + Short.valueOf(f[1]));
      z = (short)(z + Short.valueOf(f[2]));
      n = (short)(n - 1);
    }
    if (x == 0 && y == 0 && z == 0) {
      System.out.println("YES");
    }
    else {
      System.out.println("NO");
    }
  }
}

File: codeforces/low_difficulty/difficulty_1000/3.young_physicst_[69A]/young _physicist.py
# Author: José Rodolfo (jric2002)
n = int(input())
xr = yr = zr = 0
while (n):
  x, y, z = map(int, input().split(" "))
  xr += x
  yr += y
  zr += z
  n -= 1
if (xr == 0 and yr == 0 and zr == 0):
  print("YES")
else:
  print("NO")